README.rst

```diff
@@ -15,7 +15,7 @@ and answer service for BigQuery data.
    `here <http://g.co/cloud/data-qna>`_.
 
 .. |alpha| image:: https://img.shields.io/badge/support-alpha-orange.svg
-   :target: https://github.com/googleapis/google-cloud-python/blob/master/README.rst#alpha-support
+   :target: https://github.com/googleapis/google-cloud-python/blob/main/README.rst#alpha-support
```

Comment:
I get 404 with the new link. Should we migrate https://github.com/googleapis/google-cloud-python first?

Reply:
Done.